anvill/include/anvill/Program.h

```diff
@@ -162,6 +163,20 @@ struct ByteSequence {
   size_t size{0};
 };
 
+// Describes a list of targets reachable from a given source address
+struct ControlFlowTargetList final {
+	// Source address
```

Comment:
Run `scripts/format-files`; spaces for indentation! You can configure this in Eclipse. See step 6 in my document ;-)